docs/clusters/how-to-use-authorizer.md

```diff
@@ -0,0 +1,30 @@
+---
+id: what-is-authorizer
```

Comment:
The page id is "what-is-authorizer" but the file name is "how-to-use-authorizer" 

I think these should be
"what-is-an-authorizer" & "what-is-a-claim-mapper" for this and the other file.